website/source/index.html.erb

```diff
@@ -1,144 +1,434 @@
+        <p>
+          Because it knows the state of all nodes and applications, if a
+          node becomes unhealthy, Nomad will automatically reschedule the
+          applications that were previously running on the unhealthy host
+          onto a new, healthy host if capacity exists. As new clients are
+          added to the cluster, they can share the responsibility of
+          running applications and services.
+        </p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section id="flexible-workloads" class="marketing">
+  <div class="container">
+    <div class="row">
+      <div class="col-sm-12">
+        <span class="callout">Support</span>
+        <h2>Flexible Workloads</h2>
+        <p class="lead">
+          In addition to supporting Linux, Windows, and Mac, Nomad has
+          extensible support for containerized, virtualized, and standalone
+          applications. Easily start Docker containers, VMs, or application
+          runtimes like Java.
+        </p>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-sm-12">
+        <%= inline_svg "feature-flexible-workloads.svg" %>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-sm-6">
+        <h3>Diverse Technologies</h3>
+        <p>
+          By supporting a wide range of technologies such as Docker, RKT, and
+          LXC, Nomad does not force you into a single technology. Easily use
+          multiple container or virtualization runtimes simultaneously.
+          Applications with their own runtime such as Java or Golang can be
+          scheduled directly without the overhead of a container.
+        </p>
+      </div>
+      <div class="col-sm-6">
+        <h3>Multi OS</h3>
+        <p>
+          The Nomad client runs on Linux, Windows, and Mac OS. This diverse
+          operating system support enables using the same application scheduler
+          for all your scheduling and runtime needs.
+        </p>
+      </div>
+    </div>
+  </div>
+</section>
+
+
+<section id="binpacking" class="marketing black">
+  <div class="container">
+    <div class="row">
+      <div class="col-sm-12">
+        <span class="callout">Maximize</span>
+        <h2>Increase Utilization &amp; Reduce Costs</h2>
+        <p class="lead">
+          Nomad uses bin packing to optimize application placement onto servers
+          to maximize resource utilization, increase density, and help reduce
+          costs.
+        </p>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-sm-12">
+        <%= inline_svg "feature-binpacking.svg" %>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-sm-6">
+        <h3>Increase Density</h3>
+        <p>
+          Nomad places applications and services based on a bin packing
+          algorithm, which is used to optimize the resource utilization and
+          density of applications, but is also augmented by affinity and
```

Comment:
density of applications. Nomad automatically augments the bin-packing algorithm with anti-affinity rules to avoid colocating instances of the same service to reduce the probability of correlated failures.